src/shared/Core/Trace2.cs

```diff
@@ -48,6 +59,7 @@ protected override void ReleaseManagedResources()
         base.ReleaseManagedResources();
     }
 
+<<<<<<< HEAD
```

Comment:
Looks like an errant merge conflict. Maybe you take care of it in a later commit.

Reply:
I should have reviewed the commits more carefully after my last rebase. Thank you for catching.